test/model/message_test.dart

```diff
@@ -576,4 +603,197 @@ void main() {
         .reactions.isNotNull().jsonEquals([eg.unicodeEmojiReaction]);
     });
   });
+
+  group('handleSubmessageEvent', () {
+    late StreamMessage streamMessage;
```

Comment:
```suggestion
    late Message message;
```

Similar to my preceding comment (on code a bit below this) — the detail that it's a stream message doesn't matter to the narrative, so can be left out.